otel/conf.go

```diff
@@ -184,10 +184,14 @@ windowsperfcounters/mssql_{{.MSSQLID}}:
         action: update
         new_name: cpu/utilization
         operations:
-          # change label cpu -> cpu_number
-          - action: update_label
-            label: cpu
+          # take avg over cpu dimension, retaining only label: state
```

Comment:
The convention in the rest of the comments seems to be "_label_name_ label", so that would make it `retaining only state label`.

Reply:
done